database/models/measurements.js

```diff
@@ -10,8 +10,12 @@ module.exports = {
     create: function (data) {
         return databaseP.then(function (db) {
 
+        var cloneData = JSON.parse(JSON.stringify(data)); // Copy the object...
```

Comment:
Oh right! There is another way to clone an object doing;
```js
var clone = Object.assign({}, obj);
```
This is probably much more efficient than doing it via JSON.